pkgs/by-name/na/nanopb/generator-out.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv
+, cmake
+, python3
+, writeTextFile
+, protobuf
+, src
+, version
+}:
+let
+  pyproject_toml = writeTextFile {
```

Comment:
Have you considered contributing this `pyproject.toml` upstream? Apparently they used to have one so I think they'd be open to it.

Reply:
Yea I have a lot going on currently but I am planning to do so. I also want to add the cmake options to the targets as well, it is a much better way to handle their propagation than relying on package managers to handle it.